cpp_utils.py

```diff
@@ -368,14 +375,18 @@ def post_func_type_change(pfn):
         xfunc = ida_hexrays.decompile(ea)
         func_ptr_typeinf = utils.get_typeinf_ptr(xfunc.type)
         for xref in xrefs:
-            member, old_name, struct = ida_struct.get_member_by_id(xref.frm)
+            member = ida_typeinf.udm_t()
+            struct = ida_typeinf.tinfo_t()
+            struct.get_udm_by_tid(member, xref.frm)
+            if member:
+                struct = member.get_parent_struc()
```

Comment:
I think that
```suggestion
            member = ida_typeinf.udm_t()
            ida_typeinf.tinfo_t().get_udm_by_tid(member, xref.frm)
            struct = ida_typeinf.tinfo_t(tid=xref.frm)
```
is more correct, as there is no such a function named `get_parent_struc`

Reply:
ack